### PR TITLE
Added passing of kwargs to external optimizer minimize

### PR DESCRIPTION
**Pull request by Karel-van-de-Plassche**
I originally created this patch to track performance, but as it isn't an OP, you can't build timelines etc. Still it might be nice to have this interface.

### CODE REVIEW

**Comment by Karel-van-de-Plassche** [comment]
I signed it!

File: tensorflow/contrib/opt/python/training/external_optimizer.py [hunk 1]
```diff
@@ -120,6 +120,7 @@ def minimize(self, session=None, feed_dict=None, fetches=None,
         flattened into a single vector.
       loss_callback: A function to be called every time the loss and gradients
         are computed, with evaluated fetches supplied as positional arguments.
+      run_kwargs: kwargs to pass to session.run
```
Comment (joshburkart) [inline tensorflow/contrib/opt/python/training/external_optimizer.py]: Period after session.run please

**Review by joshburkart — CHANGES_REQUESTED** [review]
Looks good from my perspective

**Review by martinwicke — APPROVED** [review]
Fixed formatting, ow good.

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Comment by martinwicke** [comment]
Oddly, the test don't trigger. Trying again. 

Jenkins, test this please.

**Comment by martinwicke** [comment]
better. Thanks Jenkins.

**Pull request merged**